include/ibamr/INSVCStaggeredHierarchyIntegrator.h

```diff
@@ -140,7 +140,7 @@ class INSVCStaggeredHierarchyIntegrator : public INSHierarchyIntegrator
      * integrator object with the restart manager when the object is so
      * registered.
      */
-    ~INSVCStaggeredHierarchyIntegrator();
+    virtual ~INSVCStaggeredHierarchyIntegrator();
 
```

Comment:
Does this matter in practice?

Reply:
Not really. But in many cases in main.cpp, we have,

`Pointer<INSVCStaggeredHierarchyIntegrator> ins_integrator = new INSVCStaggeredConservativeHierarchyIntegrator(...)`

which is deleted through the base class destructor when the simulation ends. I noticed that sometimes at the end of the simulation (i.e. when main returns), I get a segfault. Making the destructor virtual fixes that, I think. This is the safe thing to do.

Reply:
Yeah, segfault will happen with this usage and without `virtual`keyword.